### ISSUE TITLE
cuhk03 with resnet50 config

### ISSUE THREAD

**Issue by Gutianpei** [jixunbo/LightMBN#7]
Hello Mr. Ji

Thanks for the amazing work and the well-implemented code! I'm trying to reproduce the result for cuhk03 dataset on resnet50 (LMBN_r) but my current parameters give ~10% performance drop compares to your result. Could you share the config file you used for LMBN_r? I guess the problem is my poor parameter tuning skills. Thanks!

**Comment by jixunbo** [comment]
Hello,
you can find below my configurations of LMBN_r for cuhk03 dataset, on which the batch size and training epochs should be 
differ from the other datasets.
Cheers

---------------------------------------
T: 3
act: relu
amsgrad: false
batchid: 8
batchimage: 8
batchtest: 32
beta1: 0.9
beta2: 0.999
bnneck: false
config: ''
cosine_annealing: false
cpu: false
cuhk03_labeled: false
cutout: false
dampening: 0
data_test: cuhk03_splited
data_train: cuhk03_splited
datadir: /content/ReIDataset/
decay_type: step_50_80_110
drop_block: false
epochs: 180
epsilon: 1.0e-08
feat_inference: after
feats: 512
gamma: 0.1
h_ratio: 0.3
height: 384
if_labelsmooth: true
loss: 0.5*CrossEntropy+0.5*MSLoss
lr: 0.00035
lr_decay: 60
margin: 0.7
model: LMBN_r
momentum: 0.9
nGPU: 1
nThread: 4
nep_id: 
nesterov: false
num_anchors: 1
num_classes: 751
optimizer: ADAM
parts: 6
pcb_different_lr: true
pool: avg
probability: 0.5
random_erasing: true
reset: false
sampler: true
test_every: 10
w_cosine_annealing: true
w_ratio: 1.0
warmup: constant
weight_decay: 0.0005
width: 128

**Comment by Gutianpei** [comment]
Thank you so much!
